app/models/competition.rb

```diff
@@ -507,7 +507,9 @@ def registration_full_and_accepted?
   end
 
   def auto_accept_threshold_reached?
-    auto_accept_disable_threshold > 0 && auto_accept_disable_threshold <= registrations.competing_status_accepted.count
+    auto_accept_disable_threshold.present? &&
+      auto_accept_disable_threshold > 0 &&
```

Comment:
```suggestion
      auto_accept_disable_threshold.positive? &&
```

Reply:
We gotta turn on that rubocop

Reply:
Updated this instance, and another 2 instances elsewhere in the file